appcues/src/main/java/com/appcues/debugger/screencapture/ViewExt.kt

```diff
@@ -0,0 +1,77 @@
+package com.appcues.debugger.screencapture
+
+import android.graphics.Rect
+import android.text.format.DateFormat
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.view.children
+import com.appcues.R
+import com.appcues.monitor.AppcuesActivityMonitor
+import com.appcues.ui.ElementSelector
+import java.util.Date
+
+internal fun View.screenCaptureDisplayName(timestamp: Date): String {
+    var name = ""
+    val activity = AppcuesActivityMonitor.activity
+    if (activity != null) {
+        name = activity.javaClass.simpleName
```

Comment:
I suppose we could try to find the current Fragment to make this suggested initial name better. I have not looked into this, just used the Activity to start with something simple.

Reply:
you may try to cast to FragmentActivity, look for supportFragment Manager maybe and see if you can gather more information, but I would not rely on that either.